notebooks/Binary Classification with the UCI Credit-card Default Dataset.ipynb

```diff
@@ -1,710 +0,0 @@
-{
```

Comment:
Besides deleting this file, also update the pointer from the landing page to point to the new example.

Reply:
Done! Pointing to https://fairlearn.org/main/auto_examples/plot_credit_loan_decisions.html which will exist after this PR.